### PR TITLE
Update deployment config

### PR DESCRIPTION
**Pull request by duanyytop**

### CODE REVIEW

File: deployment.toml [hunk 1]
```diff
@@ -1,27 +1,32 @@
...
+  "ckb-cheque-script",
+  "secp256k1_data",
+  "secp256k1_blake2b_sighash_all_dual"
+]
+
+# Replace with your own lock if you want to unlock deployed cells.
+# For example the secp256k1 lock
+[lock]
+code_hash = "0x9bd7e06f3ecf4be0f2fcd2188b23f1b9fcc88e5d4b65a8637b17723bbda3cce8"
+args = "0x0000000000000000000000000000000000000000"
```
Comment (shaojunda) [inline deployment.toml:32]: Can it be upgraded later if args is set to 0?

Comment (duanyytop) [reply deployment.toml:30]: No. The `args` is the `lock args` of the deployer.

Comment (duanyytop) [reply deployment.toml:32]: No. The deployer should update with his own `lock args` and I deployed the script with my lock args on dev chain.

Comment (shaojunda) [reply deployment.toml:32]: oh, this just an example.

Comment (duanyytop) [reply deployment.toml:32]: Maybe I can give a description here.

File: deployment.toml [hunk 2]
```diff
@@ -1,27 +1,32 @@
...
-# name = "my_dep_group"
-# cells = [
-#   "my_cell",
-#   "genesis_cell"
-# ]
+# reference to on-chain cells
+[[cells]]
+name = "secp256k1_data"
+enable_type_id = false
+location = { tx_hash = "0xf8de3bb47d055cdf460d93a2a6e1b05f7432f9777c8c474abf4eec1d4aee5d37", index = 3 }
```
Comment (shaojunda) [inline deployment.toml:12]: Do users also need to modify this? If the user needs to modify, we can also consider adding a comment.

Comment (duanyytop) [reply deployment.toml:12]: OK. I will give a comment here.

**Review by shaojunda — APPROVED** [review]

**Pull request merged**